_posts/blog/2016-09-13-code-reviews.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title: "Handbook Of Manners: How To Behave In A Code Review"
+description: Some thoughts on code reviews and the way they usually are and how they should be performed.
+modified:
+categories: blog
+author: henning_gross
+excerpt: Code reviews are a common practice for knowledge distribution, learning and increasing quality - but do we perform them in a way that actually creates value?
+tags: [Mindset, Culture, Agile, XP]
+date: 2016-09-13T17:49:00-04:00
+---
+
+In most companies __code reviews__ are part of the __process__ today. In theory they help to distribute knowledge, onboard new people and improve learning and quality.
+
+
+A code review, no matter if it is performed in a pairing manner or asynchronously, is a situation in which one person __inspects__ the work of another person.
+
+
+This can lead to the __assumption__ that the reviewer has to accept the changes the author wants to introduce and put the reviewer in a position of power over the author who has to __seek approval__.
+
+
+__Agile development teams__ strive for an identity in which everyone is __equally__ important, equally __worthy__ and in which hierarchies typically make __no difference__ on functional matters (as long as the manager doesn't have to protect the business, but that's another story).
+
+
+So what does an ideal code review look like?
+
+
+Let's start with the assumption that every reviewer has the same intent - which is to __convince__ the author to follow the suggested changes (to improve the quality of the contributed code).
+
+
+When this is the main __intent__, different approaches to code reviews can only be different __strategies__ to achieve the same goal.
+
+
+The following strategies/behavioral advice can help to achieve this goal and at the same time improve the team spirit and culture.
```

Comment:
I would just remove "strategies/behavioral", and say
> The following advice can ....

or if you want to keep it:
> The following strategies and behavioral advice can ....